pkg/server/service.go

```diff
@@ -27,10 +29,16 @@ import (
 
 type Server struct {
 	pb.UnimplementedRekorServer
+	storage *tessera.Storage
 }
 
-func (s *Server) CreateEntry(context.Context, *pb.CreateEntryRequest) (*pbs.TransparencyLogEntry, error) {
-	return nil, status.Errorf(codes.Unimplemented, "method CreateEntry not implemented")
+func NewServer(storage *tessera.Storage) *Server {
+	return &Server{storage: storage}
+}
+
+func (s *Server) CreateEntry(ctx context.Context, req *pb.CreateEntryRequest) (*pbs.TransparencyLogEntry, error) {
+	// FIXME: translate pb.CreateEntryRequest to tessera.Entry
```

Comment:
This will take longer than expected, a significant part of https://github.com/sigstore/rekor/tree/main/pkg/types needs to be reimplemented for the new protobuf object which works differently from the old types, e.g. there's no `Kind()` or `APIVersion()` any more.